photo_points/Repositories/FakeAdminReviewRepository.cs

```diff
@@ -60,11 +67,45 @@ public Capture GetCapture(long id)
             return captures.SingleOrDefault(r => r.captureID == id);
         }
 
-        //public IEnumerable<Capture> GetAllUnapproved()
-        //{
-        //   // return captures.Where(a => a.approve == true);
-        // return captures.Where(a => a.approval == Capture.ApprovalType.Approve);
 
-        //}
+        public IEnumerable<Capture> GetAllUnapproved()
```

Comment:
This can be commented out since the method was instead implemented into admin service so this is no longer being used. It should be fine to leave it uncommented though. Good work on the SaveChanges() method.